Radzen.Blazor/Common.cs

```diff
@@ -518,10 +518,10 @@ public class Query
         /// <value>The filter.</value>
         public string Filter { get; set; }
         /// <summary>
-        /// Gets or sets the filter parameters.
+        /// Gets the filter expression as a collection of filter descriptors.
         /// </summary>
         /// <value>The filter parameters.</value>
-        public object[] FilterParameters { get; set; }

```

Comment:
Not sure why you've removed FilterParameters however this will brake immediately all apps generated by Radzen IDE/Radzen Blazor Studio.

Reply:
Return.
Oh sorry, the visual studio showed that the `FilterParameters` is not used anywhere. I won't delete anything else in future commits.